agent/directory.go

```diff
@@ -300,11 +301,13 @@ func (a *Agent) SetHomeDirectory() {
 		return
 	}
 
-	usr, err := user.Current()
+	userHome, err := homedir.Dir()
 	if err != nil {
-		grip.Warning(errors.Wrap(err, "getting current user to set the home directory"))
```

Comment:
Perhaps we should keep the log for debugability? And then add another log to say we are falling back to the HOME variable (and print out the home variable too)?

Reply:
I added a log back